aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -275,6 +275,27 @@ Tensor norm_sparse(const SparseTensor& self, Scalar value) {
   return self.coalesce()._values().norm(value);
 }
 
+// --------------------------------------------------------------------
+// mv(SparseTensor, Tensor)
+// --------------------------------------------------------------------
+
+Tensor mv_sparse(const SparseTensor& self, const Tensor& vec)
+{
+  AT_ASSERT(self.is_sparse());
+
+  TORCH_CHECK(self.ndimension() == 2 && 
+              vec.ndimension() == 1,
+              "mv: two tensor dim should be 2 and 1, but got ",
+              "SparseTensor Dim: ", self.ndimension(), "Tensor Dim: ", vec.ndimension());
+
+  TORCH_CHECK(vec.size(-1) == self.size(-1),
+              "two tensor's last size should be equal");
```

Comment:
This message could be more clear. Maybe something like "expected self.size(-1) == vec.size(-1)"

Reply:
done